codespell_lib/data/dictionary.txt

```diff
@@ -8025,7 +8044,9 @@ cummunicate->communicate
 cumulatative->cumulative
 cumulattive->cumulative
 cuncurency->concurrency
-cunter->counter
+cunted->counted, hunted,
```

Comment:
This should possibly be in rare, it can be used as slang for being very drunk (as many words can).